index.rst

```diff
@@ -326,33 +326,167 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
-- The database must be available and rapidly synced to at all major data facilities (`FAFF-REQ-0055`_), analogous to what is done for the summit EFD.
+- Users require a framework/method that manages the point(s) of access, analogous to the EFD Client (`FAFF-REQ-0055`_).
+  Ideally, users will have the impression all queries are going to a single database, despite what is actually happening on the back-end(s).
+- The database must be available and rapidly synced to all major data facilities (`FAFF-REQ-0055`_), analogous to what is done for the summit EFD.
 - Summit tooling, including the Scheduler, must have immediate access to the database (`FAFF-REQ-0056`_).
 
-..
-   Plot Visualization
-   ^^^^^^^^^^^^^^^^^^^
-   Use and expansion of the plot visualization tool.
-   Also explain the current use of RubinTV
+
+The Engineering and Facilities Database (EFD)
+"""""""""""""""""""""""""""""""""""""""""""""
+
+The telemetry data from the observatory systems is recorded in the EFD.
+The data is organized by topics following the `T&S XML schema`_ and recorded in InfluxDB, a time series database optimized for queries constrained by time on large datasets.
+
+The EFD is managed by `Sasquatch`_, an RSP service that includes tools like Chronograf for data exploration and visualization and Kapacitor for alerting on the EFD data
+(see also :ref:`Metric databases in Sasquatch <metric-databases>`).
+
+.. _the-butler-registy:
+
+The Butler registry
+"""""""""""""""""""
+
+Rapid Analysis will use the Science Pipelines infrastructure for SFP.
+That infrastructure provides a Butler registry, a database for tracking processing input and output datasets.
+
+The current plan is to have a Butler repository per instrument at the Summit.
+
+Once the exposures are ingested into the Butler "raw" collection, IDs are assigned, and the registry becomes the primary source for obtaining the list of observations.
+
+The Butler registry schema is described in `DMTN-073`_.
+It is designed to support visits composed of multiple exposures.
+If the visit is composed of a single exposure, then the visit table has entries that are essentially duplicates of the science exposures in the exposure table.
+
+However, the registry database contains only a small subset of the raw image metadata (the information about the conditions under which an image was taken summarized in the image headers).
+
+Analysis outputs are not in the registry database; they are typically linked to the visit table in the registry and stored as files in the Butler datastore.
+An example of an SFP task that produces such ouput is the `ConsolidateVisitSummaryTask`_ task that persists per-detector summary statistics computed for calibrated exposures into a per-visit dataset in Parquet files.
+
+Expanding the Butler registry database to include the required raw image metadata and analysis outputs is outside the plans.
```

Comment:
I'd say something like "out of scope" rather than than "outside the plans"

Reply:
I'm seeing a thumbs up but the change isn't implemented... presumably you just haven't pushed the changes?